index.html

```diff
@@ -1,291 +1,292 @@
 
-            <ul class="skills">
-                <li>
-                    <p class="topicItem">
-                        <span>Languages</span>
-                        <span> <img class="arrow" src="./image/arrow.png" alt="'arr" /> </span>
-                    </p>
-
-                    <ul class="languages">
-                        <li>
-                            <img src="image/js.svg" alt="" />
-                            JavaScript
-                        </li>
-                        <li>
-                            <img src="image/html5.svg" alt="" />
-                            HTML
-                        </li>
-                        <li>
-                            <img src="image/css3.svg" alt="" />
-                            CSS
-                        </li>
-                    </ul>
-                </li>
-                <li>
-                    <span class="topicItem">
-                        <span>Frameworks</span>
-                        <span> <img class="arrow" src="./image/arrow2.png" alt="'arr" /> </span>
-                    </span>
-                    <hr />
-                    <div class="topicItem">
-                        <span>Skills</span>
-                        <span> <img class="arrow" src="./image/arrow2.png" alt="'arr" /> </span>
-                    </div>
-                    <hr />
-                </li>
-            </ul>
+              <span>2015</span>
+            </div>
+            <div class="description">
+              A daily selection of privately personalized reads; no accounts
+              or sign-ups required.
+            </div>
+            <div class="button-links">
+              <ul>
+                <li>html</li>
+                <li>css</li>
+                <li>javascript</li>
+              </ul>
+            </div>
+            <div class="project" >See Project</div>
+          </div>
+          <img class="img" src="image/card-4.svg" alt="" />
         </div>
-    </section>
+      </div>
+    </div>
+
+    <div id="about" class="myself">
+      <div class="left">
+        <div class="description">
+          <h2 class="topic">About <br /> Myself</h2>
+          Hello I'm a software developer! I can help you build a product ,
+          feature or website Look through some of my work and experience! If
+          you like what you see and have a project you need coded, don't
+          hestiate to contact me.
+        </div>
+        <div class="menuSection">
+          <a href="#" target="_blank">LET'S CONNECT</a>
+          <div class="socialMedia">
+            <img src="image/Twitter.png" alt="" />
+            <img src="image/Linkedin.png" alt="" />
+            <img src="image/Meduim.png" alt="" />
+            <img src="image/Github.png" alt="" />
+            <img src="image/angellist.png" alt="" />
+          </div>
+        </div>
+        <div class="project">Get my resume</div>
+      </div>
+
+      <ul class="skills">
+        <li>
```

Comment:
- Your about section looks very nice but  kindly make it match the Figma design by reducing the size of the` get my resume` button and adjust a bit the font size of your about section text :grinning: :grinning:
 check below your design compared to the Figma design :grinning: :grinning: 

## your design :-1: 
![image](https://user-images.githubusercontent.com/95297251/186886379-3c14ba81-cf4c-4e0e-a77a-5a7111f650dd.png)

## Figma design :+1: 

![image](https://user-images.githubusercontent.com/95297251/186887325-c78e2640-4afd-4a0d-8cbb-ac10662c73ef.png)